hybridse/src/udf/dynamic_lib_manager.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef HYBRIDSE_SRC_UDF_DYNAMIC_LIB_MANAGER_H_
+#define HYBRIDSE_SRC_UDF_DYNAMIC_LIB_MANAGER_H_
+
+#include <dlfcn.h>
+#include <string>
+#include <vector>
+#include <map>
+#include <mutex>
+#include "base/fe_status.h"
+
+namespace hybridse {
+namespace udf {
+
+struct DynamicLibHandle {
+    explicit DynamicLibHandle(void* ptr) {
+        handle = ptr;
+        ref_cnt = 1;
+    }
+    void* handle = nullptr;
```

Comment:
suffix `_`

Reply:
struct members need not end with '_'. refer [Variable_Names](https://google.github.io/styleguide/cppguide.html#Variable_Names)